tests/Microsoft.Identity.Web.AotCompatibiliy.TestApp/Microsoft.Identity.Web.AotCompatibiliy.TestApp.csproj

```
@@ -0,0 +1,33 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  
+  <PropertyGroup>
+    <TargetFramework>net8.0</TargetFramework>
+    <TargetFrameworks Condition="'$(TargetNet9)' == 'True'">$(TargetFramework); net9.0</TargetFrameworks>
+    <OutputType>Exe</OutputType>
+    <PublishTrimmed>true</PublishTrimmed>
+    <TrimMode>full</TrimMode>
```

```suggestion
    <PublishAot>true</PublishAot>
    <TrimmerSingleWarn>false</TrimmerSingleWarn>
```

The Wilson repo wasn't targeting net7+ when I first wrote the test app there, so it couldn't use PublishAot. But now that we are targeting net8, we should be able to use PublishAot (which has more rules than just PublishTrimmed).

Set TrimmerSingleWarn to false so each warning is reported, and not a single warning for a whole assembly.

TrimMode=full is no longer needed as of net8. That is now the default.

---

Done